shapes/bookmarks/bookmark.shex

```diff
@@ -3,6 +3,9 @@ PREFIX foaf:      <http://xmlns.com/foaf/0.1/>
 PREFIX terms:     <http://purl.org/dc/terms/>
 PREFIX xsd:       <https://www.w3.org/2001/XMLSchema#>
 
+# This shape describes some the information we store a bookmark. The
+# resource it is used describes multiple bookmarks.
+
```

Comment:
I don't know what the 2nd sentence is trying to say - is it trying to say a single Resource can have multiple bookmark instances referring to it (e.g., multiple bookmark instances with different `dcterms:created` and `dcterms:title` values, but all with the same `bookmark:recall` value)...?
```suggestion
# This shape describes some of the information we store for a bookmark. The
# resource it is used describes multiple bookmarks.

```

Reply:
The intention is to say that the resource where we use this shape for contains multiple bookmarks.

Reply:
Ah Ok - that's certainly clearer. So I'd suggest changing the 2nd sentence here to be something like this:
"Multiple instances conforming to this shape are intended to be contained together within a single resource - in other words, a single Solid resource could contain a collection of these bookmark instances, as opposed to having a dedicated Solid resource per bookmark instance."